xbmc/input/ButtonTranslator.cpp

```diff
@@ -1195,6 +1195,38 @@ CAction CButtonTranslator::GetGlobalAction(const CKey &key)
   return GetAction(-1, key, true);
 }
 
+bool CButtonTranslator::HasLonpressMapping(int window, const CKey &key)
+{
+  map<int, buttonMap>::const_iterator it = m_translatorMap.find(window);
+  if (it == m_translatorMap.end())
+  {
+    if (window > -1)
+      return HasLonpressMapping(GetFallbackWindow(window), key);
```

Comment:
Right ;)
Still this window id logic is very confusing. Shouldn't ```GetFallbackWindow(WINDOW_ADDON_START)``` return -1 to lower the confusion (with dynamic window id starting at WINDOW_ADDON_START +1)?